src/vm/debugdebugger.cpp

```diff
@@ -512,6 +517,10 @@ FCIMPL4(void, DebugStackTrace::GetStackFramesInternal,
             I4 *pILI4 = (I4 *)((I4ARRAYREF)pStackFrameHelper->rgiILOffset)->GetDirectPointerToNonObjectElements();
             pILI4[iNumValidFrames] = data.pElements[i].dwILOffset;
 
+            // Assembly
+            OBJECTREF pAssembly = pFunc->GetAssembly()->GetDomainAssembly()->GetExposedAssemblyObject(); 
```

Comment:
A nit - I've just found we can simplify this a bit as the Assembly contains GetExposedObject() method which is implemented as GetDomainAssembly()->GetExposedAssemblyObject().